### PR TITLE
Add Accessibility

### PR DESCRIPTION
**Pull request by Nelofarzabi**
In this pull request, I implemented the following project requirements:
• Check all of the items listed in the article
• Determine if your Portfolio needs any changes.
• Make changes in a separate branch and open a new pull request.
• Prepare a report listing all points that you have checked, and the result of your evaluation.
• Your report should include:
◦ List all points that you have checked
▪️ Page titles
▪️ Image text alternatives
▪️ Text headings
▪️ Color contrast
▪️ Resize
▪️ Interaction
▪️ Moving content
▪️ Multimedia
▪️ The basic structure of the page

-Page titles - no issues found except Serving images in next-gen formats
-Image text alternatives - no issues found
-Text headings -no  issues found 
-Color contrast -no  issues found 
-Resize -no issues found 
-Interaction -no issues found 
-Moving content -no issues found 
-Multimedia -no issues found 
-The basic structure of the page -no issues found

### CODE REVIEW

File: index.html
Comment (Shisui6) [inline index.html:1]: According to the project requirement, your accessibility report should include a list of all the headings in the easy checks article and the result of your evaluation. If no issues were found, it should include text stating no issues found, and if any issues were found, it should include a link to the line of code where it was found. Hence the structure of your report should look something like this:

```
- Page titles - no issues found
- Image text alternatives - no issues found
- Text headings - issues on line X and X
. . .
```

Please update your PR description to include a more comprehensive report🙏

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @Nelofarzabi and @Lawmsangi, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (LionRouge1) [inline index.html:1]: [OPTIONAL]
- [ ] Please make sure the title `<title>` of the page briefly describes the content of the page. Here is a good example of page title `<title>`: "Mar's portfolio" , "Mark full-stack webdeveloper" ...

- [ ] Kindly make sure your heading levels have a meaningful hierarchy. Ideally, the page starts has only one `h1` tag which is usually similar to the page title.

_**This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change**_

**Review by LionRouge1 — APPROVED** [review]
Hi **team**, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**